docs/guide/migrating-ntpsec.md

```diff
@@ -0,0 +1,177 @@
+# Migrating from NTPsec
+
+Both ntpsec and ntpd-rs can serve a similar role on Unix systems. This guide is aimed to help those migrating machines currently running ntpsec to ntpd-rs. We assume some experience with the [ntpsec configuration format](https://docs.ntpsec.org/latest/ntp_conf.html). A user with no or little ntpsec experience may be better of following our [getting started guide](getting-started.md).
+
+Configuration for ntpsec uses a custom format that functions as a list of commands. In contrast, ntpd-rs uses a configuration file in the `.toml` format that gives values to properties. That means that in most cases fields cannot be repeated. Comments can be added by starting them with a `#`. The remainder of the line is then ignored.
+
+An `ntpd-rs` configuration can be validated with `ntp-ctl validate -c <path>`. This will check all field names, and in some cases validates that a value is within the range of valid values for that property.
+
+This guide will not go into detail on all of ntpsec's configuration directives, but rather focus on those most important for successful migration. If a particular directive is not mentioned here, there may still be ntpd-rs options in the [configuration reference](../ntp.toml.5.md) that achieve the desired effect. Note that not all functionality of ntpsec is currently supported, a short overview of major differences is given [at the end of this document](#unsupported-features)
+
+## Time sources
+
+### Server entries
+
+The `server` and `pool` commands have a direct equivalent in ntpd-rs:
+
+```
+# ntpsec
+server 0.pool.ntp.org
+server 1.pool.ntp.org
+pool pool.ntp.org
+
+# ntpd-rs
+[[source]]
+mode = "server"
+address = "0.pool.ntp.org"
+
+[[source]]
+mode = "server"
+address = "1.pool.ntp.org"
+
+[[source]]
+mode = "pool"
+address = "pool.ntp.org"
+count = 4
+```
+
+A source in `pool` mode must explicitly define an integer `count`, the maximum number of connections from this pool. The ntpd-rs daemon will actively try to keep the pool "filled": new connections will be spun up if a source from the pool is unreachable.
+
+<!-- "broadcast" is mentioned in the docs but never defined. Maybe this is an error in their docs?
+Like ntpsec, ntpd-rs deliberately does not support symmetric and broadcasting association modes because these modes have security issues.
+-->
```

Comment:
we can just not mention these modes at all because ntpsec should not have them? I'm just confused by their one mention of broadcast 

> While there is a rich set of options available, the only required option is one or more pool, server, peer, or broadcast commands.

pool, server and peer are all defined, but broadcast isn't anywhere.